pkg/parser/v2/testdata/output/anyof.json

```diff
@@ -0,0 +1,80 @@
+{
+  "asyncapi": "2.0.0-rc1",
+  "channels": {
+    "test": {
+      "publish": {
+        "message": {
+          "payload": {
+            "anyOf": [
+              {
+                "properties": {
+                  "key": {
+                    "type": "string"
+                  }
+                },
+                "type": "object"
+              },
+              {
+                "properties": {
+                  "key2": {
+                    "type": "string"
+                  }
+                },
+                "type": "object"
+              }
+            ]
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "messages": {
+      "testMessages": {
```

Comment:
I think you don't need this message. It's not being used anywhere.

Reply:
This file is used as a `golden` to check against the output of the parser with `anyof.json` in `input` `testdata` directory.